tensorrt_llm/_torch/cute_dsl_kernels/blackwell/kimi_k3_kda/k4_persistent.py

```
@@ -0,0 +1,1489 @@
+        elif result and "ptx-options=" in result:
+            _logger.debug("k4_persistent ptx-options patch: ptx-options already present")
+            _patch_applied = True
+        return result
+
+    _CuTeDSL._get_pipeline = _patched_get_pipeline
+# Workaround for a CuTe DSL parser bug: the nvidia-cutlass-dsl 4.5.0 AST
+# preprocessor cannot parse tuple except handlers anywhere in a kernel
+# module ("'Tuple' object has no attribute 'id'"), which breaks every
+# cute.compile of this file. Keep a single bare Exception until the DSL
+# pin moves past the bug.
+except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)